src/pages/Modal/StartSimulatorModal.jsx

```diff
@@ -26,12 +28,12 @@ const InitialSettingModal = () => {
       }
     );
 
-    setIsOpenedInitial(false);
+    setIsOpenedSimulatorModal(false);
     setChuckPositionsList(initialStateArray);
   };
 
   return (
-    <Modal drection="horizontal" modalTitle="Initial Setting">
+    <Modal drection="horizontal" title="Initial Setting">
```

Comment:
```suggestion
    <Modal drection="information="">
```
초기화 기능이 제거되고 그에 따라서 모달 관련 이름들이 전체적으로 변경된 부분을 확인했습니다.
어떻게 해야하는지 안내를 하는 모달로 바뀌었다보니 위처럼 바꾸면 좀 더 직관적이지 않을까 싶습니다.

Reply:
감사합니다! 제가 미처 확인하지 못 한 부분이었네요. 조금 더 직관적으로 보이게 하기 위해 `Simulator Information`로 변경했습니다!